quantum/stm32/chconf.h

```diff
@@ -48,7 +48,7 @@
  * @details Frequency of the system timer that drives the system ticks. This
  *          setting also defines the system tick time unit.
  */
-#define CH_CFG_ST_FREQUENCY                 100000
+#define CH_CFG_ST_FREQUENCY                 1000
```

Comment:
Revert changes to this, as it's not needed, and adversely affects any timing related code on the controller.
```suggestion
#define CH_CFG_ST_FREQUENCY                 100000
```

Reply:
I assume #6301 will need the same revert?